client/src/index.jsx

```diff
@@ -152,17 +205,45 @@ class App extends React.Component {
     });
   }
 
+  handleSelectChange(e){
+    let currentValues = Object.assign({}, this.state.currentValues);
+    if(e.target.value === "15-year fixed"){
+      currentValues.years= 15
+    }else if(e.target.value === "30-year fixed"){
+      currentValues.years= 30
+    }else{
```

Comment:
space between curly brace 
```suggestion
    } else {
```